.github/workflows/ci.yml

```diff
@@ -400,7 +442,6 @@ jobs:
 
   test_tf_integrations_gpu:
     needs: [setup, build_all]
-    if: fromJson(needs.setup.outputs.should-run)
```

Comment:
Do we also need to revert this before submit?

Reply:
Yes